packages/core/src/index.ts

```diff
@@ -842,10 +910,6 @@ class Hopr extends EventEmitter {
     verbose('Stopping heartbeat & indexer')
     this.heartbeat?.stop()
 
-    verbose('Stopping database')
-    await this.db?.close()
```

Comment:
So the closing of the DB is done implicitly?

Reply:
It is not, the DB should be init and closed from the TS code.

Reply:
@NumberFour8